ccmlib/common.py

```diff
@@ -492,12 +492,10 @@ def check_socket_available(itf, return_on_error=False):
         s.bind(sockaddr)
         s.close()
         return True
-    except socket.error as msg:
+    except socket.error:
         s.close()
-        if return_on_error:
-            return False
         addr, port = itf
-        raise UnavailableSocketError("Inet address %s:%s is not available: %s" % (addr, port, msg))
+        raise UnavailableSocketError("Inet address %s:%s is not available; a cluster may already be running or you may need to add the loopback alias" % (addr, port))
```

Comment:
Nit: I'd keep the msg param, just for completeness (in case the reason for failure is something different):

```
"Inet address %s:%s is not available (%s); a cluster may already be running or you may need to add the loopback alias" % (addr, port, msg)
```